### ISSUE TITLE
make config parameters consistent across all steps

### ISSUE THREAD

**Issue by erflynn** [UCSF-DSCOLAB/data_processing_pipelines#33]
Currently `pre_qc` uses different parameters, we want to make these all consistent so folks can use one config for multiple steps.
Highest priority to make `pre_qc` and `post_qc` consistent; `pre_fmx_qc` and `post_fmx_qc` can come later.

**Comment by amadeovezz** [comment]
>  I do not think the libraries should be called dir

I originally used the word `dir` since this value is injected in path values -  as the directory where the fastq files live. 

My thought was, "as a user I should be able to point to a directory where my fastqs live"

I now see that `library name = dir`, which was confusing to me at first (and I think still is for new comers)

For this iteration, I think it is okay to change this key to `name`, given that this is a convention we use in the lab. In the future - I'd like to make the input file even more intuitive, and I think it is worth revisiting.

**Comment by erflynn** [comment]
good point, very open to updating the config to make it more intuitive.
